src/js/common/schemaform/ObjectField.jsx

```diff
@@ -149,6 +149,9 @@ class ObjectField extends React.Component {
     // description and title setup
     const showFieldLabel = uiSchema['ui:options'] && uiSchema['ui:options'].showFieldLabel;
     const title = uiSchema['ui:title'] || schema.title;
+    const CustomTitleField = typeof title === 'function'
```

Comment:
Can you also add this to the schemaform form-config doc? This way we know `ui:title` can accept a react component.